### DISCUSSION TITLE
University of Helsinki: Full Stack open 2022 - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#221] [Announcements]
# University of Helsinki: Full Stack open 2022 - CS自学指南

CS自学指南

https://csdiy.wiki/Web%E5%BC%80%E5%8F%91/fullstackopen/

**Comment by barbarianxs** [comment]
冲

**Reply by guojianchen** [reply]
冲不到的

**Comment by charlie1587** [comment]
提交系统打不开啊

**Comment by liminmsn** [comment]
web开发这一个模块太潦草了，一点不详细

**Reply by azonips314** [reply]
是啊，还是得看react和express的文档做补充

**Comment by xls138** [comment]
还有没有这种类型的,我挺喜欢的,不太喜欢看视频

**Comment by Rcpentuy** [comment]
我想制作多人在线小游戏，请问这门课合适吗

**Comment by thewangcj** [comment]
学到 part4，对我这样有一点 web 基础的人来说非常好，尤其是课后作业非常棒，量大管饱且难度合适，总体上推荐给有一点前端基础的同学。

**Comment by Rcpentuy** [comment]
如果看的是中文版建议时不时check一下英语版。中文版有很多内容没有及时更新。比如一些语法细节，关于ai编程工具使用的建议等等

**Comment by nullsir06** [comment]
有没有一起学的

**Reply by liminmsn** [reply]
学什么

**Reply by CraftOldWang** [reply]
你好可以一起学吗，qq：3973037287

**Reply by nullsir06** [reply]
忘记写联系方式了(汗)，q:2010402073

**Comment by EnzoDing-rgb** [comment]
没视频，通过教程学习，还不错
